shells/chrome/src/panel.js

```diff
@@ -114,10 +121,4 @@ function reload() {
   }, 100);
 }
 
-// chrome.devtools.panels added in Chrome 18.
-// chrome.devtools.panels.themeName added in Chrome 54.
-const themeName = (chrome.devtools.panels : any).themeName === 'dark'
-  ? 'ChromeDark'
-  : 'ChromeDefault';
-
-ReactDOM.render(<Panel alreadyFoundReact={true} themeName={themeName} {...config} />, node);
+ReactDOM.render(<Panel alreadyFoundReact={true} {...config} />, node);
```

Comment:
Should the `alreadyFoundReact` prop be added after the `config` prop? Otherwise, `alreadyFoundReact` will always be false as defined on line 30.